scripts/finetune.py

```diff
@@ -163,6 +166,82 @@ def check_not_in(list1: List[str], list2: Union[Dict[str, Any], List[str]]) -> b
     return not any(el in list2 for el in list1)
 
 
+class HttpHandler(http.server.BaseHTTPRequestHandler):
+    def __init__(self, *args, cfg=None, prompter=None, tokenizer=None, model=None, **kwargs):
+        self.cfg = cfg
+        self.prompter = prompter
+        self.tokenizer = tokenizer
+        self.model = model
+        super().__init__(*args, **kwargs)
+
+    def do_POST(self):
+        content_length = int(self.headers['Content-Length'])
+        post_data = self.rfile.read(content_length)
+        instruction = post_data.decode('utf-8')
+        
+        if not instruction:
+            response = ""
+        else:
+            default_tokens = {"unk_token": "<unk>", "bos_token": "<s>", "eos_token": "</s>"}
```

Comment:
we could probably grab these from the tokenizer.

Reply:
Patches welcome.  :)  Again, this is just redoing a copy of the preexisting inference section as a webserver. I didn't change the logic.